### PR TITLE
Change border colors on the minimap for moved units

### PR DESCRIPTION
**Pull request by mangofeet**

### CODE REVIEW

**Review by Windchild292 — APPROVED** [review]

**Comment by mangofeet** [comment]
There are some color blindness issues in the Green-Blind/Deuteranopia and Monochromacy/Achromatopsia categories that I should sort out before this is merged

File: megamek/src/megamek/client/ui/swing/minimap/Minimap.java [hunk 1]
```diff
@@ -999,17 +1009,17 @@ private void paintUnit(Graphics g, Entity entity) {
...
 
             // Fill the form in player color / team color
             g.setColor(iconColor);
             g2.fill(form);
             
             // Black border
-            g2.setColor(Color.BLACK);
-            g2.setStroke(new BasicStroke(8f, BasicStroke.CAP_BUTT, BasicStroke.JOIN_MITER));
+            g2.setColor(borderColor);
+            g2.setStroke(new BasicStroke(innerBorderWidth/2, BasicStroke.CAP_BUTT, BasicStroke.JOIN_MITER));
```
Comment (Windchild292) [inline megamek/src/megamek/client/ui/swing/minimap/Minimap.java:1022]:
```suggestion
            g2.setStroke(new BasicStroke(innerBorderWidth / 2f, BasicStroke.CAP_BUTT, BasicStroke.JOIN_MITER));
```

**Review by Windchild292 — CHANGES_REQUESTED** [review]
Just one spacing issue, which should probably be a float too

**Review by Windchild292 — APPROVED** [review]
Looks good, thanks for the PR

**Pull request merged**